examples/README.md

```diff
@@ -0,0 +1,13 @@
+# Homebase React Examples
+
+You can clone this repo to have ready to run [homebase-react](https://github.com/homebaseio/homebase-react) examples.
+
+## Examples
+[Counter Example](counter-example/)
```

Comment:
I think we should rename to `counter` given these all live in the `examples/` folder. And the same goes for all the other examples.